tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs

```diff
@@ -2150,6 +2156,8 @@ public void TestGroupWithKeyFunction()
         [SkippableFact]
         public void TestGroupWithMaxTime()
         {
+#pragma warning disable 618
```

Comment:
This works! We tend use wrap only the deprecated method call with the `#pragma` statements though, like so: 
```
#pragma warning disable 618
                        Assert.Throws<MongoExecutionTimeoutException>(() => _collection.Group(args));
#pragma warning restore
```

This wasn't in our dev/style guide, so I've updated the guide to indicate this :)